CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.15.0
+### Added
+- Support for project type `pimcore` in project type resolvers.
```

Comment:
Please also update the README.md on that pimcore is now also a supported product type